hnn_core/tests/test_parallel_backends.py

```diff
@@ -204,7 +204,7 @@ def test_mpi_failure(run_hnn_core_fixture):
     assert "MPI processes are unable to reach each other" in stdout
 
     expected_string = "Child process failed unexpectedly"
-    assert len(record) == 1
-    assert record[0].message.args[0] == expected_string
+    # last warning is the expected one, earlier ones DeprecationWarnings
```

Comment:
Ideally you want to remove the deprecated warning in the tests by doing what we ask users to do in the future ...

Reply:
duh! fixed in 468f2b3